Sources/WebUI/WebViewProxy.swift

```diff
@@ -92,6 +92,14 @@ public final class WebViewProxy: ObservableObject {
         webView?.wrappedValue.load(request)
     }
 
+    /// Loads the contents of the specified HTML string and navigates to it.
+    /// - Parameters:
+    ///   - string: The string to use as the contents of the webpage.
+    ///   - baseURL: The base URL to use when the system resolves relative URLs within the HTML string.
+    public func loadHTMLString(_ string: String, baseURL: URL?) {
+        webView?.wrappedValue.loadHTMLString(string, baseURL: baseURL)
+    }
```

Comment:
Is it difficult to add UI tests for this method?

Reply:
@b1ackturtle I did it.
[0ba2d5a](https://github.com/cybozu/WebUI/pull/25/commits/0ba2d5a8b25ac689ed24f0d5e901a1f200520fe0)

Reply:
https://github.com/cybozu/WebUI/actions/runs/11365353567
Passed the test.